app/javascript/data/models/playbooks.js

```diff
@@ -0,0 +1,22 @@
+import { array, string, object } from 'yup';
+
+export const serviceTemplatePlaybookSchema = array()
+  .of(
+    object().shape({
+      href: string().required(),
+      id: string().required(),
+      name: string().required()
+    })
+  )
+  .nullable();
+
+export const PLAYBOOK_JOB_STATES = {
+  PENDING: 'pending',
+  ACTIVE: 'active',
+  FINISHED: 'finished'
+};
+
+export const PLAYBOOK_JOB_STATUS = {
+  OK: 'Ok',
+  ERROR: 'Error'
+};
```

Comment:
Not a critical thing, but, from the pre/post playbooks email thread

> Because we rely on Service Templates, each playbook invocation is in fact a service provision request. So we have an miq_request object, so we made the choice that job_state and job_status are respectfully miq_request.request_state and miq_request.state

While the email states that these are provision requests, I think the statuses are the same for the `ServiceTemplateTransformationPlanRequest` model, so if we name these more generically, we could reuse them

Reply:
Yea - i wasn't sure if those would mimic plan statuses/states exactly, but agree it appears they will. I was hoping to make use of those globally as well going forward. Moved the playbook job statuses/state exports to the plan and we can adjust later if needed.